test/integration/physics_system.cc

```diff
@@ -33,6 +34,7 @@
 
 #include "ignition/gazebo/Server.hh"
 #include "ignition/gazebo/SystemLoader.hh"
+#include "ignition/gazebo/Util.hh"
```

Comment:
alphabetize ?

Reply:
From my understanding, I thought that upper case (in this case, `Util`) comes before lowercase (`test_config`) because of how case ordering works in the [ASCII table](https://www.asciitable.com/) (upper case comes before lower case). However, I still haven't found a definitive guide/resource that confirms this. Does anyone else know how alphabetization should be handled for different-case letters?

Reply:
Maybe checking how popular tools like clang-format are implemented could give a direction? My usual configuration keeps the order as it is right now, i.e. with the lower case headers below the upper case one.

Reply:
> Maybe checking how popular tools like clang-format are implemented could give a direction?

It looks like there are [several options](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) in clang-format (see the section titled `SortIncludes`), and I'm not sure which one is default. I'll stick to `CaseSensitive` since that is what I have seen elsewhere.

> Btw, having a project-wise style configuration would be a dream becoming true :smile: 

We actually had a discussion about this today in our internal simulation meeting. We use [`cppcheck`](https://github.com/danmar/cppcheck/) which catches some styling, but not all. We discussed using something like clang, but @chapulina mentioned that she's tried this before, and that it took quite some time to run (I believe it took a few hours to complete :exploding_head:). So, if you have any thoughts about style configurations we could use, we're open to suggestions :smile: we can open a separate issue to discuss this.